test/hotspot/jtreg/runtime/cds/DeterministicDump.java

```diff
@@ -79,6 +79,7 @@ static String dump(ArrayList<String> args, String... more) throws Exception {
         String archiveName = logName + ".jsa";
         String mapName = logName + ".map";
         CDSOptions opts = (new CDSOptions())
+            .addPrefix("-Xint") // Override any -Xmixed/-Xcomp flags from jtreg -vmoptions
```

Comment:
I assume this issue could pop up in newer tests as well. Is there a place where you can specify that `-Xint` needs to be explicitly added when doing a static dump?

Reply:
-Xint is necessary only when we want to test for deterministic dumps. We don't have many such test cases.